Clients/GUI/Scripts/SignUper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using GUI.GUIModels;
+using System.Net;
+using DTO;
+using System.IO;
+using System.Text.Json;
+
+namespace GUI.Scripts
+{
+    public class SignUpper
+    {
+        public static HttpStatusCode SignUp(SignUpViewModel input)
+        {
+            var output = new UserEmailPassword(input.Email, input.Password);
+
+            var request = (HttpWebRequest)WebRequest.Create("https://localhost:5011/user/create");
+            request.Method = "POST";
+            string jsonOutput = JsonSerializer.Serialize(output);
+            request.ContentType = "application/json";
+            using (var dataStream = new StreamWriter(request.GetRequestStream()))
+            {
+                dataStream.Write(jsonOutput);
+            }
+
+            var response = (HttpWebResponse)request.GetResponse();
+            return response.StatusCode;
```

Comment:
We can do it like that: 
```
if (response.IsSuccessStatuscode)
{
// do sth 
}
else
{
//throw some exception
}
```
And then we will handle that in GUI code

Reply:
I'm not sure whether it's the best option, but in another pull request Spartak mentioned that returning `HttpStatusCode` is not a good practice